components/tabs/tab-panel-mixin.js

```diff
@@ -33,11 +33,23 @@ export const TabPanelMixin = superclass => class extends superclass {
 		this.role = 'tabpanel';
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+
+		changedProperties.forEach((oldVal, prop) => {
+			if (prop === 'selected') {
+				if (this.selected) {
+					requestAnimationFrame(() => {
+						this._dispatchSelected();
+					});
+				}
+			}
+		});
+	}
+
 	async attributeChangedCallback(name, oldval, newval) {
 		super.attributeChangedCallback(name, oldval, newval);
-		if (name === 'selected') {
-			if (this.selected) this._dispatchSelected();
-		} else if (name === 'text') {
+		if (name === 'text') {
```

Comment:
In `master`, I'll move this to `updated` too, but keeping the cert fix more selective